CRD-DTR/HomeOxygenTherapy/R4/files/HomeOxygenTherapyRule-0.1.0.cql

```diff
@@ -4,12 +4,13 @@ include FHIRHelpers version '4.0.0' called FHIRHelpers
 
 parameter Patient Patient
 parameter device_request DeviceRequest
+parameter Coverage Coverage
 
 define "Age":
     AgeInYears()
 
 define RULE_APPLIES:
-    "Age" > 0
+    "Age" > 0 and Coverage.class.value = 'Medicare Part A'
```

Comment:
This changes it so that Teddy Roosevelt should no longer satisfy this condition since his coverage in test-ehr has been changed to Medicare Part B. It does not effect any other patients since they're all on Medicare Part A. Where should we publish this?